tensorflow/README.md

```diff
@@ -0,0 +1,69 @@
+# Sample running TensorFlow models from Hazelcast Jet
+
+TensorFlow is a popular library to train and use machine learning
+models. We integrate it with Jet to classify stream of events with the
+result of a TF model execution.
+
+TensorFlow provides two ways of running models:
+- In-process: load the model into memory and execute it
+- In ModelServer: a request-response based service that executes the
+model and can also handle multiple versions of the models etc. The calls
+can be made using gRPC or through a REST API.
+
+Each way has pros and cons and discussion of them is beyond the scope of
+this sample.
+
+As a sample, we took the Large Movie Reviews Dataset as provided by the
+TensorFlow Keras Datasets. We trained a model as described in a
+[tutorial](https://www.tensorflow.org/tutorials/keras/basic_text_classification)
+and save it. The result of the training is in the
+[`data/model`](data/model) subdirectory.
+
```

Comment:
Could we please put a few lines about what is the actual classification done here along with the input and output types ?